qds_sdk/pipelines.py

```diff
@@ -339,23 +339,70 @@ def create(cls, pipeline_name, create_type, **kwargs):
 
         Args:
             pipeline_name: Name to be given.
-            create_type: 1->Assisted, 2->Code, 3->Jar
+            create_type: 1->Assisted, 2->Jar, 3->Code
```

Comment:
hmm, breaking change, need to remember while releasing

Reply:
This was wrongly specified. I have corrected this.
create_type was always 2 for Jar and 3 for Code.

Reply:
okay got it